kitchen.dokken.yml

```
@@ -20,27 +20,52 @@ platforms:
       pid_one_command: /bin/systemd
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
-
-  - name: centos-6
+  
+  - name: debian-10
     driver:
-      image: dokken/centos-6
-      pid_one_command: /sbin/init
+      image: dokken/debian-10
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        - RUN /usr/bin/apt-get update
 
   - name: centos-7
     driver:
       image: dokken/centos-7
       pid_one_command: /usr/lib/systemd/systemd
 
-  - name: ubuntu-16.04
+  - name: centos-8
     driver:
-      image: dokken/ubuntu-16.04
-      pid_one_command: /bin/systemd
-      intermediate_instructions:
-        - RUN /usr/bin/apt-get update
+      image: dokken/centos-8
+      pid_one_command: /usr/lib/systemd/systemd
 
   - name: ubuntu-18.04
     driver:
       image: dokken/ubuntu-18.04
       pid_one_command: /bin/systemd
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
+
+  - name: fedora-31
+    driver:
+      image: dokken/fedora-31
+      pid_one_command: /usr/lib/systemd/systemd
+  
+  - name: fedora-32
+    driver:
+      image: dokken/fedora-32
+      pid_one_command: /usr/lib/systemd/systemd
+
+  - name: oracle-7
+    driver:
+      image: dokken/oraclelinux-7
+      pid_one_command: /usr/lib/systemd/systemd
+
+  - name: oracle-8
+    driver:
+      image: dokken/oraclelinux-7
+      pid_one_command: /usr/lib/systemd/systemd
+
+  - name: amazonlinux-2
+    driver:
+    image: dokken/amazonlinux-2
+    pid_one_command: /usr/lib/systemd/systemd
```

The indentation is incorrect here and causing the issue.
```suggestion
      image: dokken/amazonlinux-2
      pid_one_command: /usr/lib/systemd/systemd
```